### ISSUE TITLE
build-flo

### ISSUE THREAD

**Issue by steeevieee** [grogg/build-scripts#1]
One of the early parts of the build-flo script is to cd into "cd $PLATFORM_DIR/kernel" but kernel is a file not a dir. Am I missing something ?

**Comment by grogg** [comment]
Yes, I updated the source to include the prebuilt kernels. I did not edit
the scripts yet. Typically, I build all three devices and I just chose the
build-all-nok.sh. I will update the scripts when I have time.
